src/tests/fetchMany.test.ts

```diff
@@ -0,0 +1,39 @@
+import fetchMany from '../fetchMany';
+
+import test from 'tape';
+
+/** Get the SHA-256 hash of an ArrayBuffer */
```

Comment:
I would move this to either `utils` or `tests/helpers/sha256.ts`

Reply:
FWIW, sha512 is faster on 64 bit architectures, like travis 👿